runtime/Java/src/org/antlr/v4/runtime/BufferedTokenStream.java

```diff
@@ -209,7 +209,7 @@ public Token get(int i) {
 		if ( start<0 || stop<0 ) return null;
 		lazyInit();
 		List<Token> subset = new ArrayList<Token>();
-		if ( stop>=tokens.size() ) stop = tokens.size()-1;
+		if ( stop>=size() ) stop = size()-1;
```

Comment:
:exclamation: This is part of a bounds check for directly indexing `tokens`. The `tokens.get(i)` expression that follows needs to be updated to `this.get(i)` so the `size()` and `get(int)` methods are paired.